driver/src/main/java/org/neo4j/driver/v1/Config.java

```diff
@@ -67,6 +72,7 @@ private Config( ConfigBuilder builder )
 
         this.encryptionLevel = builder.encruptionLevel;
         this.trustStrategy = builder.trustStrategy;
+        this.authToken = builder.authToken;
```

Comment:
The reason we didn't add authToken to config was that that makes having a default config kind-of pointless unless we have add a defaultConfigWithAuthToken factory method.